src/sentry/static/sentry/app/views/alerts/rules/details/index.tsx

```diff
@@ -77,17 +95,28 @@ class AlertRuleDetails extends React.Component<Props, State> {
     return {
       start,
       end,
+      period,
       label: timeOption.label as string,
-      period: timePeriod,
     };
   }
 
   fetchData = async () => {
-    this.setState({isLoading: true, hasError: false});
     const {
+      api,
       params: {orgId, ruleId},
+      location,
     } = this.props;
 
+    this.setState({isLoading: true, hasError: false});
+
+    fetchOrgMembers(api, orgId);
```

Comment:
do we need to call this function in `componentDidMount()` and here?

Reply:
we don't need to here would be better to remove it